spec.html

```diff
@@ -2699,7 +2699,7 @@ <h1>Property Attributes</h1>
                 *undefined*
               </td>
               <td>
-                If the value is an Object it must be a function object. The function's [[Call]] internal method (<emu-xref href="#table-additional-essential-internal-methods-of-function-objects"></emu-xref>) is called with an empty arguments list to retrieve the property value each time a get access of the property is performed.
+                If the value is an Object it must be a function object. The function's [[Call]] internal method (<emu-xref href="#table-additional-essential-internal-methods-of-function-objects"></emu-xref>) is called with an empty arguments List to retrieve the property value each time a get access of the property is performed.
```

Comment:
Don't capitalize list here. This paragraph is referring to a conceptual argument list, I'm pretty sure, since it's talking about calling the method as you might in JS code.